src/xtgeo/common/sys.py

```diff
@@ -216,7 +215,7 @@ def __init__(
         if isinstance(filelike, str):
             filelike = pathlib.Path(filelike)
 
-        self._file = filelike
+        self._file: pathlib.Path | io.BytesIO | io.StringIO = filelike
```

Comment:
is there a reason why we don't use the FileLike defined in common.types here ?

Reply:
It includes str, in this case some code futher up the chain converts str to Path.